docs/source/config/intro.rst

```diff
@@ -112,6 +112,18 @@ At present, this only affects the current session - changes you make to
 config are not saved anywhere. Also, some options are only read when
 IPython starts, so they can't be changed like this.
 
+.. _configure_embedded:
+
+Configuring embedded IPython
```

Comment:
Doesn't embed() do the same thing as start_ipython, except it grabs the global and local namespaces automatically?
Anyway, since the file is entirely to do with configuration, a heading of "Starting IPython from Python" would be sufficiently obvious. Other candidates could be something all the lines of "Using a Python caller" or "Configuring directly from caller code" but I think "Starting IPython from Python" makes the most sense. 

Also, doesn't the first code block under the embedding section of the [IPython reference](https://ipython.readthedocs.io/en/stable/interactive/reference.html#embedding-ipython) use start_ipython?